sormas-backend/src/main/java/de/symeda/sormas/backend/externalsurveillancetool/ExternalSurveillanceToolGatewayFacadeEjb.java

```diff
@@ -86,14 +85,15 @@ public void sendEvents(List<String> eventUuids, boolean archived) throws Externa
 	private void sendRequest(ExportParameters params) throws ExternalSurveillanceToolException {
 		String serviceUrl = configFacade.getExternalSurveillanceToolGatewayUrl().trim();
 
-		Response response = ClientBuilder.newBuilder()
-			.connectTimeout(30, TimeUnit.SECONDS)
-			.build()
-			.target(serviceUrl)
-			.path("export")
-			.request()
-			.post(Entity.json(params));
-		int status = response.getStatus();
+//		Response response = ClientBuilder.newBuilder()
```

Comment:
Why are these changes in the test (commented out code)?

Reply:
committed by mistake